source/core/GenericFunctions.cpp

```diff
@@ -1048,9 +1146,23 @@ InstructionCore* EmitGenericInRangeAndCoerceInstruction(ClumpParseState* pInstru
 
             pInstructionBuilder->BeginEmitSubSnippet(&snippetBuilder, vectorBinOp, snippetArgId);
 
-            snippetBuilder.EmitInstruction(&savedOperation, 7, xEltType, (void*)nullptr, loEltType, (void*)nullptr, hiEltType, (void*)nullptr,
-                                           booleanType, (void*)nullptr, booleanType, (void*)nullptr,
-                                           coercedEltType, (void*)nullptr, destEltType, (void*)nullptr);
+            snippetBuilder.EmitInstruction(
+                &savedOperation,
+                7,
+                xEltType,
+                nullptr,
+                loEltType,
+                nullptr,
+                hiEltType,
+                nullptr,
+                booleanType,
+                nullptr,
+                booleanType,
+                nullptr,
+                coercedEltType,
+                nullptr,
+                destEltType,
+                nullptr);
```

Comment:
Oh, c'mon.
The surrounding code is somewhat complex, and it's much more beneficial to be able to see it all on the screen at once then to break these out on new lines  What is the point?